scopes/lanes/ui/overview/card/lane-component-card.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import { ComponentCard } from '@teambit/explorer.ui.gallery.component-card';
+import { PreviewPlaceholder } from '@teambit/preview.ui.preview-placeholder';
+import { LoadPreview } from '@teambit/workspace.ui.load-preview';
+import { LaneComponentModel } from '@teambit/lanes.lanes.ui';
+import styles from './lane-component-card.module.scss';
+
+export type LaneComponentCardProps = {
+  component: LaneComponentModel;
+} & React.HTMLAttributes<HTMLDivElement>;
+
+export function LaneComponentCard({ component, ...rest }: LaneComponentCardProps) {
```

Comment:
isn't this.. `WorkspaceComponentCard` ? if it's intended for the same thing with just `overrideInternalLink`, maybe compose that and pass through that prop?
or better yet, use the `href` prop